model_analyzer/config/input/config_command_profile.py

```diff
@@ -1197,3 +1203,16 @@ def _autofill_values(self):
 
             new_profile_models[model.model_name()] = new_model
         self._fields['profile_models'].set_value(new_profile_models)
+
+    def _using_request_rate(self) -> bool:
```

Comment:
didn't we already have similar code in a different story? Any way to reuse (or store the result somewhere)?

Reply:
Yes, we do. The logic is very similar, but I don't see how it can be reused. It's in `perf_analyzer_config_generator.py:167` in the branch `add-support-for-request-rate-range`

Reply:
this function is in ConfigCommandProfile . A ConfigCommandProfile is passed into PerfAnalyzerConfigGenerator, right? Can PACG leverage this new code in CCP instead of figuring it out on its own?

Also, am I reading it correctly on line 1216 that if ANY model has request rate enabled that this will return true? Won't that break if there is a mix?  Can this function take a model name to fix that?

Reply:
I've added a code and unit testing to check for this case. Please re-review.